src/containers/ConversionForm/index.tsx

```diff
@@ -36,6 +37,12 @@ interface ConversionProps {
     setError: ActionCreator<SetErrorAction>;
 }
 
+interface ExtensionMap {
+    [key: string]: string;
```

Comment:
this should be typed as `[key: AvailableEngines]`

Reply:
also this interface should be in `state/trajectory/conversion-engine-types`

Reply:
Ok , moving to proper folder.

Typescript doesn't like `[key: AvailableEngines]`, doesn't an object key have to be a string?

Tried this `[key in AvailableEngines]: string` , but also getting errors.

Will keep poking around.

Reply:
example: 
```TypeScript
export const ENGINE_TO_TEMPLATE_MAP: { [key in AvailableEngines]: string } = {
    [AvailableEngines.Smoldyn]: "smoldyn_data",
    [AvailableEngines.Cytosim]: "cytosim_data",
    [AvailableEngines.CellPack]: "cellpack_data",
    [AvailableEngines.SpringSalad]: "springsalad_data",
};
```

Reply:
what's the error you're getting?

Reply:
```TypeScript 
export type ExtensionMap = {
    [key in AvailableEngines]: string;
}
```